GiEnJul/ClientApp/src/components/admin/common/RecipientTable.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from "react";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Avatar,
+  Container,
+  Divider,
+  Typography,
+} from "@material-ui/core";
+import {
+  ExpandMore,
+  Group,
+  Phone,
+  Mail,
+  CheckRounded,
+  CloseRounded,
+} from "@material-ui/icons";
+import { RecipientType } from "../overview/Recipient";
+import useStyles from "./Styles";
+
+type Props = {
+  data: RecipientType[] | [];
+};
+
+const DatatableRecipient: React.FC<Props> = ({ data }) => {
+  const classes = useStyles();
+
+  const formatFamily = (input: Number) => {
+    if (input < 3) {
+      return "< 3";
+    }
+    if (input > 5) {
+      return "> 5";
+    } else {
+      return "3 - 5";
+    }
+  };
+
+  const handleMatched = (input: Boolean) => {
+    if (input) {
+      return classes.matched;
+    } else {
+      return classes.notMatched;
+    }
+  };
+
+  const handleGender = (gender: Number, age: Number) => {
```

Comment:
The gender type in common/enums/Genders.tsx should be used here instead of Number. This applies to other places as well.